src/utils/collection/index.ts

```diff
@@ -195,7 +195,20 @@ export const isSystemCollection = (type: CollectionSummaryType) => {
 export const shouldShowOptions = (type: CollectionSummaryType) => {
     return !OPTIONS_NOT_HAVING_COLLECTION_TYPES.has(type);
 };
-
+export const showTrashQuickOption = (type: CollectionSummaryType) => {
+    return type === CollectionSummaryType.trash;
+};
+export const showDownloadQuickOption = (type: CollectionSummaryType) => {
+    return !(type === CollectionSummaryType.trash);
+};
+export const showShareQuickOption = (type: CollectionSummaryType) => {
+    return !(
+        type === CollectionSummaryType.trash ||
```

Comment:
can say when to show it... The current logic will start showing it for any new type, we add by default. 

if we decide where to show it then... rather than where to not... I will never to mistakenly shown for cases we didn't intend to